### PR TITLE
Daily PR June 3

### PR DESCRIPTION
**Pull request by pe7r**

### CODE REVIEW

File: src/Components/Header/Header.js [hunk 1]
```diff
@@ -1,19 +1,52 @@
...
 
 export default class Header extends Component {
+    state = {
+        activatePopup: false,
+        setPopup: ''
+    }
+
+    handlePopupChange = (id) => {
+        this.setState({
+            activatePopup: true,
```
Comment (ruslanpalagin) [inline src/Components/Header/Header.js]: prefer to name boolean vars starting with "is". Example: isPopupActive

File: src/Components/Header/Header.js [hunk 2]
```diff
@@ -1,19 +1,52 @@
...
+            setPopup: id
```
Comment (ruslanpalagin) [inline src/Components/Header/Header.js]: Naming pattern "setSOMETHING" "getSOMETHING" are commonly using for methods. 
For this var use could use "selectedPopupId"

File: src/Components/Loader/Loader.scss [hunk 3]
```diff
@@ -0,0 +1,75 @@
+html, body {
```
Comment (ruslanpalagin) [inline src/Components/Loader/Loader.scss]: VERY dangerous. Avoid setting global styles on some component

File: src/Components/Loader/Loader.scss [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    height: 100%;
+  }
+
+#cube-loader {
```
Comment (ruslanpalagin) [inline src/Components/Loader/Loader.scss]: Copy pasted?!
You could do it better, but OK))

File: src/Components/SearchInput/SearchInput.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react'
```
Comment (ruslanpalagin) [inline src/Components/SearchInput/SearchInput.js]: Please use functional comp.

File: src/Components/Header/Header.js (hunk 1 above)
Comment (pe7r) [reply src/Components/Header/Header.js]: Done.

File: src/Components/Header/Header.js (hunk 2 above)
Comment (pe7r) [reply src/Components/Header/Header.js]: Done.

File: src/Components/Loader/Loader.scss (hunk 3 above)
Comment (pe7r) [reply src/Components/Loader/Loader.scss]: Deleted.

File: src/Components/Loader/Loader.scss (hunk 4 above)
Comment (pe7r) [reply src/Components/Loader/Loader.scss]: Do you want me to do it by myself?)

File: src/Components/SearchInput/SearchInput.js (hunk 5 above)
Comment (pe7r) [reply src/Components/SearchInput/SearchInput.js]: Done.

**Pull request merged**